src/kernel-install/60-ukify.install.in

```diff
@@ -171,6 +171,17 @@ def kernel_cmdline(opts) -> str:
     return ' ' + ' '.join(options)
 
 
+def initrd_list(opts) -> list[Path]:
+    initrds = opts.initrd
+    microcode = list(opts.staging_area.glob('microcode/*'))
+    initrds = microcode + initrds
+
+    initrd = opts.staging_area / 'initrd'
+    if initrd.is_file():
+        initrds.append(initrd)
```

Comment:
~~`initrds += [initrd]` should be used for consistency with the rest of the code.~~